### PR TITLE
Snow Map Generator

### PR DESCRIPTION
**Pull request by Azrun**
## About the PR 
Adds Snow Map Generator
https://streamable.com/jeqwov

- https://streamable.com/x3luz6 (Outdated)
- https://streamable.com/iv6aqp (Outdateder)

Biomes Added:

- Desert (Based on Underwater Space)
- Snow
- Snow Rocky
- Snow Forest
- Snow Forest Thick
- Snow Rough
- Water Clear
- Water Ice (Slippy)

We have Trenchify, we have Swampify, why not... Winterify

## Why's this needed? 
I want this for _Artemis_.
May be fun to land the station on an ice planet?

## Changelog

```changelog
(u)Azrun and Walpvrgis
(+)Graphical assets for snow experiences and future snow planets...
```

### CODE REVIEW

**Comment by virvatuli-gh** [comment]
I love it when a plan comes together.

File: code/modules/admin/randomverbs.dm [hunk 1]
```diff
@@ -2752,6 +2752,36 @@ var/global/mirrored_physical_zone_created = FALSE //enables secondary code branc
...
+	if(src.holder.level >= LEVEL_ADMIN)
+		switch(alert("Turn space into a snowscape? This is probably going to lag a bunch when it happens and there's no easy undo!",,"Yes","No"))
+			if("Yes")
+				var/image/I = new /image/ambient
+				var/datum/map_generator/snow_generator/map_generator = new
+				var/list/space = list()
+				for(var/turf/space/S in block(locate(1, 1, Z_LEVEL_STATION), locate(world.maxx, world.maxy, Z_LEVEL_STATION)))
+					space += S
+				map_generator.generate_terrain(space)
+				for (var/turf/S in space)
```
Comment (Sovexe) [inline code/modules/admin/randomverbs.dm:2775]:
```suggestion
				for (var/turf/S as anything in space)
```

**Review by Sovexe — COMMENTED** [review]
just about everything looks ok to me

**Comment by Gannets** [comment]
Cool idea!

**Comment by Azrun** [comment]
@WALPVRGIS Mind taking a look at the current sprites and let me know if there are any sprites you want to add/update/remove?

**Comment by Azrun** [comment]
Snow will be a follow up for atomic PRs.

**Pull request merged**